packages/styles/scss/_mixins.scss

```diff
@@ -403,3 +403,11 @@
     }
   }
 }
+
+@mixin hero-grid-rows($max-height: $spacing-hero-image-max-height) {
+  grid-template-rows:
+    minmax($max-height, 1fr)
```

Comment:
Isn't this actually min-height? 😄

Reply:
I was thinking the same xD, I didnt want to change the naming conventions they have already done so i was sticking to it but i'll refactor it

Reply:
They have a variable defined as $spacing-hero-image-max-height

Reply:
If the existing names can be improved, we should go for it. 😄

in this case, if $spacing-hero-image-max-height is always referring to a min-height, then go ahead and rename it and then this param.

Reply:
Made the refactor of the variable